### ISSUE TITLE
CSS Styles Menu Misconfiguration or Bug?

### ISSUE THREAD

**Issue by vscode-pmaher** [tinymce/tinymce#6143]
**What is the current behavior? Describe the bug**
CSS styles menu displays 'Paragraph' by default on toolbar

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**
 Here is config:
`  tinymce.init({
		    selector: '#tiny',
        menubar:false,
        'content_css': 'my-styles.css',
        importcss_append: false,
        plugins: ['importcss'],
	toolbar: 'undo redo searchreplace | paste pastetext | formatselect | styleselect '
			});`

**What is the expected behavior?**
I want the CSS styles menu to display as  'Styles' by default it seems counter-intuitive to have two menu both showing 'Paragraph' by default and I don't want the CSS styles appended to the format menu.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
TinyMCE v5.5.0

**Comment by lostkeys** [comment]
Can you tell me more what you want to achieve? The `styleselect` and `formatselect` sort of do the same thing, so having them both seems weird to us.

**Comment by vscode-pmaher** [comment]
Hi,

I've probably not explained this very well, basically I don't understand why the styleselect menu shows 'paragraph' at the top of the menu by default? I'd expect it to say 'Styles' or something like that. Unless it is a misconfiguration on my part? I basically have two menus on the toolbar for formatselect  nd styleselect and both say 'Paragraph' by default

**Comment by lostkeys** [comment]
Both formatselect and styleselect shows the currenly used [format](https://www.tiny.cloud/docs/configure/content-formatting/#formats). The default format is the "Paragraph", matching the `<p>` tag basically.

The styleselect doesn't have anything to do with CSS style really, think of it more in terms of visual appearances. The differences between styleselect and formatselect are mainly in the config options available:
* The `formatselect` button is configured by the [block_formats option](https://www.tiny.cloud/docs/configure/editor-appearance/#block_formats) and is rather basic.
* The `styleselect` button is configured through the [style_formats option](https://www.tiny.cloud/docs/configure/editor-appearance/#style_formats) and is much more advanced. It can contain nested menus, have groups etc.

**Comment by vscode-pmaher** [comment]
get it now I thought styleselect was a dedicated menu for css styles from external stylesheet... I think I need to create a single custom menu that incorporate basic formatting and styles from my stylesheet, that will be the best option

thanks

**Comment by lostkeys** [comment]
I don't exactly know what you are trying to solve, but I would create custom formats that assigns relevant classes or inline styles and then simply assign those to the styleselect via [style_formats option](https://www.tiny.cloud/docs/configure/editor-appearance/#style_formats). [Look here for inspiration](https://www.tiny.cloud/docs/demo/format-custom/)

Cheers!

**Comment by jackkemmish** [comment]
Hi there, can I open this back up as I have a similar concern?

I noticed in TinyMCE 6 the custom `style_formats` default title is `Paragraph` but previously it was `Formats`. Is this configurable as my custom `style_formats` doesn't actually have any paragraph styles, `Formats` was better here as it's more abstract.

Example before:
![Screenshot 2023-11-14 at 17 05 37](https://github.com/tinymce/tinymce/assets/10498947/f0b14e41-8d6b-45b1-9d92-8820a7da71bb)

Example now:
![Screenshot 2023-11-14 at 17 07 47](https://github.com/tinymce/tinymce/assets/10498947/98973a32-2853-429d-8eba-95f06e65de9c)

Taking the screenshot, I just noticed the elements title is still `Formats` too.

Thanks!

Jack
